### PR TITLE
Create a json file

### LINKED ISSUE #6799: Hi, add me. :)
Hello! :wave: Please add me as a collaborator to this repo.

### PR DESCRIPTION
**Pull request by Maicco**
I'm learning how to use github.
@githubteacher 
closes #6799

### CODE REVIEW

**Comment by Maicco** [comment]
Commit

File: _pins/Maicco.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+githubHandle: Maicco
```
Comment (crichID) [inline _pins/Maicco.json]: @Maicco you are very close! You only need to add three dashes at the beginning and end of your file so it looks like this:

```
---
githubHandle: Maicco
latitude: -23.5733
longitude: -46.6417
---
```

**Comment by crichID** [comment]
👋 @Maicco I left a comment to help you finish this. If you need help, please check out this video: https://vimeo.com/201011288

**Comment by brianamarie** [comment]
@Maicco  I made some changes so the tests will pass. Once everything turns green, I'll go ahead and merge this in since it's been a few days and we want to keep the repository clean. 😄 ✨

**Pull request merged**